spec.md

```diff
@@ -1386,6 +1398,8 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 
 #### `NodeGetId`
 
+`NodeGetId` RPC call is deprecated. Users of this RPC call SHOULD use `NodeGetInfo`.
```

Comment:
nit: different sentences should be on different lines

Reply:
In spec.md - it is all in one line. github is wrapping the line. But still - I do not under why different sentences should start on different lines. We start new sentence on same line all the time when writing a paragraph don't we?

Reply:
we generally try to follow the rules documented here: https://github.com/container-storage-interface/spec/blob/master/CONTRIBUTING.md